docs/learn-teleport.md

```diff
@@ -0,0 +1,41 @@
+---
+id: learn-teleport
+title: Teleporting Assets To Other Parachains
+sidebar_label: Teleporting Assets
+---
+
+One of the main properties Polkadot and Kusama brings to ecosystem is blockchain interoperability.
+Teleportation is the process of moving assets, such as coins, tokens, or NTFs, to other chains
```

Comment:
Would be good if we can relate teleportation with interoperability - so because of interoperability -> we can teleport, and it would be helpful to mention the characteristics of teleporting itself - "burning on one chain, minting on the other"